.vortex/cli/src/Command/AbstractInstallCommand.php

```
@@ -0,0 +1,420 @@
+        success: 'Files copied to destination directory',
+      );
+
+      Task::action(
+        label: 'Preparing demo content',
+        action: fn(): string|array => $this->fileManager->prepareDemo($this->getFileDownloader()),
+        success: 'Demo content prepared',
+      );
+    }
+    catch (\Exception $exception) {
+      Tui::output()->setVerbosity(OutputInterface::VERBOSITY_NORMAL);
+      Tui::error('Installation failed with an error: ' . $exception->getMessage());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexCli\Utils\Tui' in method 'doInstall'. (undefined)

(StaticAccess)